Library/Homebrew/cmd/upgrade.rb

```diff
@@ -3,7 +3,7 @@
 #:
 #:    Options for the `install` command are also valid here.
 #:
-#:    If `--cleanup` is specified or `HOMEBREW_UPGRADE_CLEANUP` is set then remove
+#:    If `--cleanup` is specified, `HOMEBREW_INSTALL_CLEANUP` or `HOMEBREW_UPGRADE_CLEANUP` is set then remove
```

Comment:
As a clarification around the original feature, is the intent to keep both ENVs or to enhance what the old one did as well a rename it?

Reply:
@Wojonatior Good question 👍 Let's remove the documentation for `HOMEBREW_UPGRADE_CLEANUP` in favour of `HOMEBREW_INSTALL_CLEANUP` but keep the `HOMEBREW_UPGRADE_CLEANUP` variable functional for people who are still using it.